waiter/resources/log4j.properties

```diff
@@ -6,15 +6,15 @@ log4j.category.waiter=INFO
 log4j.appender.InfoAppender=org.apache.log4j.DailyRollingFileAppender
 log4j.appender.InfoAppender.Threshold=INFO
 log4j.appender.InfoAppender.File=log/${waiter.logFilePrefix}waiter.log
-log4j.appender.InfoAppender.DatePattern='.'yyyy-MM-dd
```

Comment:
Should it be only `HH` for hourly?

Reply:
Yep, doing on minutes was for testing purposes